Source/Scene/Camera.js

```diff
@@ -3243,5 +3259,27 @@ define([
      * @callback Camera~FlightCancelledCallback
      */
 
+    function _limitOrientation(angle, camera) {
```

Comment:
You don't need to use an underscore for the name here.  That would only be if the function was on the object, ie `Camera.prototype._privateFunction` or `Camera._privateFunction`